lightning/src/ln/channelmanager.rs

```diff
@@ -3353,6 +3384,12 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		// during initialization prior to the chain_monitor being fully configured in some cases.
 		// See the docs for `ChannelManagerReadArgs` for more.
 		let _persistence_guard = PersistenceNotifierGuard::new(&self.total_consistency_lock, &self.persistence_notifier);
+
+		assert_eq!(*self.last_block_hash.read().unwrap(), header.block_hash(),
```

Comment:
Oops, probably best to drop the test commit, looks like its still failing the fuzzer, we can take it in a separate PR.

Reply:
Odd, was passing locally. Will drop the commit for now.